app/models/atlas.rb

```diff
@@ -33,7 +33,7 @@ def places
   # All of the atlas places where there is a ListedTaxon demonstrating presence
   def presence_places
     exploded_place_ids_to_include, exploded_place_ids_to_exclude = get_exploded_place_ids_to_include_and_exclude
-    descendant_listed_taxa = ListedTaxon.joins( list: :check_list_place ).
+    descendant_listed_taxa = ListedTaxon.joins( list: :place ).
```

Comment:
I should also point out that this change (and related changes below) might cause serious problems, but they didn't seem to cause any specs to fail.

Reply:
Am I understanding this correctly that if there is a comprehensive list for Phrymaceae in Nevada County, and the Atlas editor lists Erythranthe lewisii in Nevada County, this change will only create a listing on the comprehensive list, but not on the default checklist?

Reply:
Partially. Currently, if there is a comprehensive list for Phrymaceae in Nevada County, the atlas editor will not let you add Nevada County to the atlas for Erythranthe lewisii. You are correct in saying that this change will only create a listing on the comprehensive list, not the default checklist. But we could change that if you want.

Reply:
my preference would be for the taxon in question (e.g. Erythranthe lewisii) to get added to the default list. All of the atlas and comprehensive stuff to date only deals with the default lists and I think its likely there's unforeseen bugs and complications if they also have to be aware of non-default checklists